### DISCUSSION TITLE
Installation error under windows 10

### DISCUSSION THREAD

**Discussion by socger** [pimcore/pimcore#11434] [Q&A / Support] [answered]
I work in a xamp environment under windows 10

After modifying php.ini to put the extensions that PIMCORE needs, I tried to install the demo project with this command ...
`composer create-project pimcore/demo my-project`

But at the end of Composer installation I get the following exception...
```
Script Pimcore\Composer::clearCache handling the pimcore-scripts event terminated with an exception

 [ErrorException]
  Array to string conversion

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--ask] [--] [<package>] [<directory>] [<version>]

```

To be honest I don't know how to solve the problem and I would appreciate your help.
Thanks in advance.

**Comment by ruccvcs50x** [comment, marked as answer]
go to your my-project/vendor/pimcore/pimcore/lib/Composer.php

Find for function : executeCommand

change: $process = new Process($command, null, null, null, $timeout);
to : $process = new Process(implode(" ", $command), null, null, null, $timeout);

then cd my-project and run composer up
